templates.go

```diff
@@ -115,7 +115,7 @@ func getTemplates() *template.Template {
 	{{ if .SignInMessage }}
 	<p>{{.SignInMessage}}</p>
 	{{ end}}
-	<button type="submit" class="btn">Sign in with a {{.ProviderName}} Account</button><br/>
+	<button type="submit" class="btn">Sign in with an OpenShift account</button><br/>
```

Comment:
should we substitute Openshift only if .ProviderName is not set ?

Reply:
Maybe, but would we ever have a different provider?

Reply:
It is possible people use another Oauth server in front of openshift for SSO, in that case they may want to specify to the user that those are the credentials being used/requested ?
Doesn't hurt to have the capability.

Reply:
That wouldn't change the provider name, oauth-proxy is only aware that it is using an "OpenShift" type provider by way of some URLs. Any more information about the OAuth server is doing is not exposed in any way that I know of..

Reply:
ah, ok, sad.

Reply:
I think at this point we only want OpenShift.

Also, at some point we need to restyle this page (the UI team would do that for us and make it consistent with Jenkins oauth page).  Can you reach out to @spadgett and discuss when he might have time to do that (post 3.7 probably)?